client/my-sites/media-library/filter-bar.jsx

```diff
@@ -125,15 +125,14 @@ export class MediaLibraryFilterBar extends Component {
 		return (
 			<SectionNavTabs>
 				{ tabs.map( filter => (
-					<FilterItem
+					<SectionNavTabItem
 						key={ 'filter-tab-' + filter }
-						value={ filter }
 						selected={ this.props.filter === filter }
-						onChange={ this.changeFilter }
+						onClick={ this.changeFilter( filter ) }
```

Comment:
Hmm, only downside to this is we're recreating these on each render, when we know we have a fixed list of filters that don't change. Digging through code a bit this mostly translates to a state update and stat bump.

```jsx
	onFilterChange = filter => {
		if ( filter !== this.state.filter ) {
			analytics.mc.bumpStat( 'editor_media_actions', 'filter_' + ( filter || 'all' ) );
		}

		this.setState( { filter } );
	};
```

We probably had the child component do the following, which avoids the extra closure above.
```
this.props.onChange( this.props.value );
```

@Copons I think this is a reasonable approach, but do you mind doing a quick render count check to make sure the tabs behaving well?

Reply:
We're talking about ~30 renders at first load + ~3 renders every filter change.
I don't think this is _that_ bad perf wise. If anything, I'd probably seek to reduce the 30 initial renders.